podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -249,6 +249,28 @@ def __getitem__(self, index):
         else:
             return StackedCoordinates([c[index] for c in self._coords])
 
+    def __setitem__(self, dim, c):
+        if not dim in self.dims:
+            raise KeyError("Cannot set dimension '%s' in StackedCoordinates %s" % (dim, self.dims))
+
+        # try to cast to ArrayCoordinates1d
+        if not isinstance(c, BaseCoordinates):
+            c = ArrayCoordinates1d(c)
+
+        if c.name is None:
+            c.name = dim
+
+        if dim in self.dims:
+            idx = self.dims.index(dim)
+            coords = list(self._coords)
+            coords[idx] = c
+
+            self._check_sizes([c.size for c in coords])
+            self._check_names([c.name for c in coords])
+            self._check_coord_ref_sys(coords)
+
```

Comment:
I think this is why I didn't implement it from the beginning... I didn't know whether it would ever be useful to change the StackedCoordinates coordinates for one dimension if you could only do so without changing the size, and it's a bit tricky to set it up to allow you change the size by setting one dimension at a time and then only checking the size, etc afterwards.

Reply:
To me, it makes a lot of sense that you can only change the size if you change the whole stacked coordinates (i.e. `stack_coords['lat_lon'] = ... `) This would still work because its happening at the `Coordinates` level.  If you change only one dim of a stacked coordinate, you have to check that its consistent